### DISCUSSION TITLE
[QUESTION] Is event sourcing applicable for this type of solution?

### DISCUSSION THREAD

**Discussion by karlbal** [jasontaylordev/CleanArchitecture#403] [Q&A]
I really like this solution template and used it for my personal project. I already implemented two separate solutions in one project, one for the main application and the other one is for messaging using SignalR and used DotNetCore.Cap (Rabbitmq) for event publishing for handling user profile changes. 

This is just out of curiosity though as we all know, microservice is trending these days. And I'm thinking of committing to this architecture. I understand that it's not really necessary to implement event sourcing for microservices, but I think it gives a lot of advantages especially with data availability. I'm still new to this kind of topic, so I really appreciate any inputs. 

Thank you very much!

**Comment by pcarvsilva** [comment]
I think you my have problems with Persistence into two different dbs, if you could solve that problem I think it would be ok

**Reply by Eagleleader** [reply]
It is possible to use as many different DBs as you need using different DBContexts (I'm actually doing it) inside the clean architecture.
You can also use different DBContext for a DB.
